js/packages/openinference-vercel/src/VercelSemanticConventions.ts

```diff
@@ -0,0 +1,108 @@
+import { ValueOf } from "./typeUtils";
+
+/**
+ * Below are the semantic conventions for @vercel/otel instrumentation.
```

Comment:
Just a it but I don't think these attributes are baked into @vercel/otel - they are baked into the libraries themselves like the ai SDK no?

Reply:
that's a good question, those functions are baked into the libraries themselves but i'm not sure if that's where these attributes are defined and set. Seems like it would happen in @vercel/otel since that's where the "instrumentation" presumably is

Reply:
It's baked in: https://github.com/vercel/ai/blob/main/packages/ai/core/telemetry/record-span.ts